src/wallet/coinselection.cpp

```diff
@@ -35,6 +35,7 @@ struct {
  * inputs, plus the amount the selection exceeds the spending target:
  *
  * waste = selectionTotal - target + inputs × (currentFeeRate - longTermFeeRate)
+ * 
```

Comment:
Please don't add an empty line here. Also there is trailing space on this line tripping up the linter.

Reply:
ah sorry that was my IDE